extension/popup/popup.css

```diff
@@ -196,7 +196,7 @@ body,
 }
 
 .results-largeText {
-  font-size: 72px;
+  font-size: 1em;
```

Comment:
I don't think we want to change this. Instead we should use something else (besides `largeText`) above